apps/judicial-system/web/src/components/PdfButton/PdfButton.tsx

```diff
@@ -57,8 +57,16 @@ const PdfButton: React.FC<Props> = ({
   ) : (
     <Box
       data-testid={`${pdfType || ''}PDFButton`}
-      className={styles.pdfRow}
-      onClick={handleClick ? handleClick : pdfType ? handlePdfClick : undefined}
+      className={`${styles.pdfRow} ${disabled ? '' : styles.cursor}`}
+      onClick={
+        disabled
+          ? undefined
+          : handleClick
+          ? handleClick
+          : pdfType
+          ? handlePdfClick
+          : undefined
+      }
```

Comment:
Consider making this more readable? 
Maybe just having if sentences,  for example : 

```
onClick={ () => {
if(!disabled) {
 handleClick()
}else {
...... 
}

} }
```